stories/LoadingSpinner.stories.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import LoadingSpinner from '../src/components/LoadingSpinner';
+import {createDocsPage} from './utils/createDocsPage';
+
+export default {
+  title: 'Components/LoadingSpinner',
+  component: LoadingSpinner,
+  parameters: {
+    docs: {
+      page: createDocsPage()
+    }
+  }
+};
+
+export const LoadingSpinnerWithoutMessage = () => {
+  return <LoadingSpinner />;
+};
+
+export const LoadingSpinnerWithMessage = args => {
+  return <LoadingSpinner message={args.message ?? 'Loading...'} />;
+};
```

Comment:
With Storybook Controls, in most cases, I don't think we really need multiple stories for the different options of a component. Just one story with the `message` control should suffice.
Here's how I'd make that change:
```suggestion
export const Default = args => {
  return <LoadingSpinner {...args} />;
};
Default.storyName = 'LoadingSpinner';
```
Since there's only one story we can take advantage of single story hoisting by calling this Default then set the storyName to the name of the component. This will tell Storybook to display the story itself in the sidebar instead of a folder containing the story.

Reply:
Ahh super cool, ill make those adjustments!